src/building/suggested.md

```diff
@@ -63,6 +63,33 @@ Task] instead:
 
 [Build Task]: https://code.visualstudio.com/docs/editor/tasks
 
+## Configuring `ctags`
+
+One of the challenges with rustc is that the RLS can't handle it, since it's a
+bootstrapping compiler. This makes code navigation difficult. One solution is to
+use `ctags`.
```

Comment:
This is no longer relevant since RA is suggested right above.

Actually, I wonder if it still makes sense to suggest CTags at all. RA works decently with vim as far as I know, I would like to avoid suggesting multiple different IDEs if possible.

Reply:
That's what I was thinking. Maybe removing it completely makes more sense.

Reply:
@nikomatsakis @mark-i-m do you still use ctags at all? If so, do you think they make sense to recommend now that rust-analyzer is a thing?

Reply:
My primary use of ctags is for the [vim tagbar](https://github.com/majutsushi/tagbar.git) pluggin, and RA doesn't currently work with that tmk... there are other pluggins that do, I think, but I haven't spent a lot of time looking.

Reply:
I do not use ctags anymore, I'm all in on VSCode + rust-analyzer at this point.

Reply:
I'd be ok with removing discussion of this hack.

Reply:
> My primary use of ctags is for the vim tagbar pluggin, and RA doesn't currently work with that tmk... there are other pluggins that do, I think, but I haven't spent a lot of time looking.

Gotcha. I think it's ok to leave that out here - people using vim *and* RA *and* the tagbar is probably not super common, and there's still docs at https://github.com/universal-ctags/ctags in case people are still using it.